DATA/production/qc-async/trditstpc.json

```
@@ -0,0 +1,100 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "type": "2",
+        "number": "42"
+      },
+      "monitoring": {
+        "url": "influxdb-unix:///tmp/telegraf.sock"
+      },
+      "consul": {
+        "url": "alio2-cr1-hv-aliecs:8500"
+      },
+      "conditionDB": {
+        "url": "alio2-cr1-hv-qcdb1.cern.ch:8083"
+      }
+    },
+    "tasks": {
+      "Digits": {
+        "active": "true",
+        "taskName": "Digits",
+        "className": "o2::quality_control_modules::trd::DigitsTask",
+        "moduleName": "QcTRD",
+        "detectorName": "TRD",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource_comment": "no comment",
+        "dataSource": {
+          "type": "direct",
+          "query": "digits:TRD/DIGITS;tracklets:TRD/TRACKLETS;triggers:TRD/TRKTRGRD"
+        },
+       "taskParameters": {
+          "peakregionstart": "7.0",
+          "peakregionend": "20.0",
+          "pulseheightpeaklower": "1.0",
+          "pulseheightpeakupper": "5.0"
+        }
+      },
+      "Tracklets": {
+        "active": "true",
+        "taskName": "Tracklets",
+        "className": "o2::quality_control_modules::trd::TrackletsTask",
+        "moduleName": "QcTRD",
+        "detectorName": "TRD",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource_comment": "no comment",
+        "dataSource": {
+          "type": "direct",
+          "query": "digits:TRD/DIGITS;tracklets:TRD/TRACKLETS;triggers:TRD/TRKTRGRD"
+        },
+        "taskParameters": {
+          "peakregionstart": "7.0",
+          "peakregionend": "20.0",
+          "pulseheightpeaklower": "1.0",
+          "pulseheightpeakupper": "5.0"
+        }
+      },
+      "PulseHeightTrackMatch": {
+        "active": "true",
+        "className": "o2::quality_control_modules::trd::PulseHeightTrackMatch",
+        "moduleName": "QcTRD",
```

Not sure if we risk overlapping task names in this case, but you may want to consider this
```suggestion
      "TRDPulseHeightTrackMatch": {
        "active": "true",
        "taskName": "PulseHeightTrackMatch",
        "className": "o2::quality_control_modules::trd::PulseHeightTrackMatch",
        "moduleName": "QcTRD",
```

---

but then this should be done for all tasks no ? I thought the name before the { of the task is the taskname ?

---

done

---

We clarified this on MM, but for the record, the key is taskID, which should be unique throughout all the tasks of all the detectors. `taskName` is what you want e.g. in your QCDB path (`qc/TRD/MO/PulseHeightTrackMatch`), if it's not specified, the taskID is used.

I asked Sean to change it because the naming convention requires not using detector name in the task name. In other words, the 2nd `TRD` would be redundant: `qc/TRD/MO/TRDPulseHeightTrackMatch`